fili-core/src/test/groovy/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProviderSpec.groovy

```diff
@@ -138,86 +132,31 @@ class LuceneSearchProviderSpec extends SearchProviderSpec<LuceneSearchProvider>
         1 * keyValueStore.put('cardinality_key', '14')
     }
 
-    def "moveDirEntries moves all entries of a directory to a new directory, while keeping all old empty dirs"() {
```

Comment:
moved to `UtilsSpec.groovy`, because the method under this test has been moved to `Utils.java`